lib/stat_tracker.rb

```diff
@@ -21,8 +21,8 @@ def initialize(locations)
     @game_teams = csv_data(locations[:game_teams], GameTeam)
     @game_stats = GameStats.new(@games)
     @league_stats = LeagueStats.new(@games, @teams, @game_teams)
-    # @season_stats = SeasonStats.new(@games, @game_teams, @teams)
-    # @team_stats = TeamStats.new(@games, @game_teams, @teams)
+    @season_stats = SeasonStats.new(@games, @game_teams, @teams)
+    @team_stats = TeamStats.new(@games, @game_teams, @teams)
```

Comment:
Uncommented these to prepare for adding Iteration 4 and 5 stat methods

Reply:
smart